services/self-service/src/main/java/com/epam/dlab/backendapi/resources/ProjectResource.java

```diff
@@ -214,4 +217,18 @@ public Response updateBudget(
 		projectService.updateBudget(projects);
 		return Response.ok().build();
 	}
+
+	@Operation(summary = "Generate keys for project", tags = "project")
+	@ApiResponses({
+			@ApiResponse(responseCode = "202", description = "Keys are successfully generated")
+	})
+	@POST
+	@Path("/generateKeys")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@RolesAllowed("/api/project")
+	public Response generate(@Parameter(hidden = true) @Auth UserInfo userInfo) {
+		return Response
+				.accepted(keyService.generateKeys(userInfo))
```

Comment:
It is better to create DTO used by controller with e.g. publicKey and privateKey fields